src/app/services/views/ServiceListTabsView.vue

```diff
@@ -9,33 +9,38 @@
     <div
       class="stack"
     >
-      <div v-html="t('services.routes.items.intro', {}, { defaultMessage: '' })" />
       <AppView>
         <template #actions>
-          <XActionGroup
-            :expanded="true"
-          >
-            <template
-              v-for="{ name } in route.children"
-              :key="name"
+          <div class="service-types-action-group">
+            <XActionGroup
+              :expanded="true"
             >
-              <XAction
-                v-if="!(!can('use service-insights', props.mesh) && ['service-list-view', 'external-service-list-view'].includes(name))"
-                :class="{
-                  'active': route.child()?.name === name,
-                }"
-                :to="{
-                  name,
-                  params: {
-                    mesh: route.params.mesh,
-                  },
-                }"
-                :data-testid="`${name}-sub-tab`"
+              <template
+                v-for="{ name } in route.children"
+                :key="name"
               >
-                {{ t(`services.routes.items.navigation.${name}`) }}
-              </XAction>
-            </template>
-          </XActionGroup>
+                <XAction
+                  v-if="!(!can('use service-insights', props.mesh) && ['service-list-view', 'external-service-list-view'].includes(name))"
+                  :class="{
+                    'active': route.child()?.name === name,
+                  }"
+                  :to="{
+                    name,
+                    params: {
+                      mesh: route.params.mesh,
+                    },
+                  }"
+                  :data-testid="`${name}-sub-tab`"
+                >
+                  {{ t(`services.routes.items.navigation.${name}.label`) }}
+                </XAction>
+              </template>
+            </XActionGroup>
+
+            <p
```

Comment:
Ah just noticed, this tag will have to be a `div v-html`.

markdown will always wrap a sentence with a `p` so you are gonna get double `p`s here if you use `p v-html`.

Good rule of thumb here is always use `div v-html` when dealing with i18n markdown.

Reply:
Ah, good point 👍 thank you 🙂